evap/staff/tests/test_importers.py

```diff
@@ -206,9 +236,11 @@ def create_existing_course(self):
         return existing_course, existing_course_evaluation
 
     def test_valid_file_import(self):
-        success_messages, warnings, errors = EnrollmentImporter.process(
-            self.default_excel_content, self.semester, None, None, test_run=True
-        )
+        excel_content = excel_data.create_memory_excel_file(excel_data.test_enrollment_data_filedata)
+        import_args_withoutDates = [excel_content, self.semester, None, None]
+        import_args_withDates = [excel_content, self.semester, self.vote_start_datetime, self.vote_end_date]
```

Comment:
This must be the famous `snake_has_eatenPascalCase` convention.

Please go with consistent snake case: `import_args_without_dates` and `import_args_with_dates`